test-data/src/test/resources/org/openmrs/module/fhir2/providers/createUpdate.xml

```diff
@@ -0,0 +1,6 @@
+<Task xmlns="http://hl7.org/fhir">
```

Comment:
Is this file actually used anywhere?

Reply:
Ohh yeah this is part of xml file in relation to json file